CHANGELOG-1.1.md

```diff
@@ -1,16 +1,19 @@
 # Changelog since v1.0.1
 
 ## Breaking Changes
+
```

Comment:
@humblec how about adding a linter to for md? https://github.com/markdownlint/markdownlint

Reply:
I think that would be a great idea so that we can enforce it on every pull request. Can you look into enhancing https://github.com/kubernetes-csi/csi-release-tools?

Reply:
sure.. I can :) .. thanks ! but this should be good to go here . @msau42